### ISSUE TITLE
[Web] RuntimeError: memory access out of bounds

### ISSUE THREAD

**Issue by XQSDD** [microsoft/onnxruntime#16984]
### Describe the issue

I'm trying to run the model in Vue using onnxruntime-web@1.15.1, I have no problem running the project directly in the compiler, but when I package the project and put it into nginx for publishing, running the model 'this.model.run(feeds)' gives me the error RuntimeError: memory access out of bounds error. I've tried a lot of ways on the internet but I can't solve it, I don't know if it's a problem with onnxruntime-web, because each version reports the error in a different location.

### To reproduce

const initModel = async () => {
      try {
        if (this.MODEL_DIR === undefined) return
        const URL = this.MODEL_DIR
        this.model = await ort.InferenceSession.create(URL)
      } catch (e) {
        console.log(e)
      }
    }
initModel()
const results = await this.model.run(feeds, option)

### ONNX Runtime Installation

Released Package

### ONNX Runtime Version or Commit ID

1.15.1

### Execution Provider

'wasm'/'cpu' (WebAssembly CPU)

**Comment by simon987** [comment]
I'm having the same problem with Vue. It works in development mode, but when compiling with `--mode production` I see `memory access out of bounds`.

I managed to "fix" it by disabling minification of unused code:

```javascript
// vue.config.js
module.exports = {
    // [...]
    configureWebpack: config => {
        config.optimization.minimizer = [new TerserPlugin({
            terserOptions: {
                compress: {
                    unused: false,
                }
            }
        })]
    }
}
```

**Comment by XQSDD** [comment]
> 我在 Vue 上遇到了同样的问题。它在开发模式下工作，但是在使用编译时，我看到.`--mode production``memory access out of bounds`
> 
> 我设法通过禁用未使用代码的缩小来“修复”它：
> 
> ```js
> // vue.config.js
> const TerserPlugin = require("terser-webpack-plugin");
> 
> module.exports = {
>     // [...]
>     configureWebpack: config => {
>         config.optimization.minimizer = [new TerserPlugin({
>             terserOptions: {
>                 compress: {
>                     unused: false,
>                 }
>             }
>         })]
>     }
> }
> ```

Thank you very much for your answer, I tried it and solved the problem, thank you very much!!!!

**Comment by rusiaaman** [comment]
I got it fixed by upgrading the webpack version from '5.73.0' to '5.88.2'
